### PR TITLE
typo fixes

### PR DESCRIPTION
**Pull request by knotts14**
#### Problem
incorrect spelling and typos

#### Summary of Changes
corrected incorrect spelling and typos

### CODE REVIEW

File: accounts-db/src/accounts_db/geyser_plugin_utils.rs [hunk 1]
```diff
@@ -121,7 +121,7 @@ impl AccountsDb {
 
             // later entries in the same slot are more recent and override earlier accounts for the same pubkey
             // We can pass an incrementing number here for write_version in the future, if the storage does not have a write_version.
-            // As long as all accounts for this slot are in 1 append vec that can be itereated olest to newest.
+            // As long as all accounts for this slot are in 1 append vec that can be iterated olest to newest.
```
Comment (steviez) [inline accounts-db/src/accounts_db/geyser_plugin_utils.rs]: nit: Since you're already in here, wanna fix "olest" to "oldest"

**Review by steviez — COMMENTED** [review]
Saw one more typo in the diff that we can fix; once that is done, looks good to me

File: accounts-db/src/accounts_db/geyser_plugin_utils.rs (hunk 1 above)
Comment (knotts14) [reply accounts-db/src/accounts_db/geyser_plugin_utils.rs]: fixed! good eyes lol

**Review by steviez — APPROVED** [review]
LGTM - thank you for the fixes!

The changes are all in comments so going to push without running CI

**Pull request merged**